ocaml/xapi/xapi_vm_snapshot.ml

```diff
@@ -250,8 +250,8 @@ let copy_vm_fields ~__context ~metadata ~dst ~do_not_copy ~default_values =
   List.iter
     (fun (key,value) ->
        let value =
-         if List.mem_assoc key default_values
-         then List.assoc key default_values
+         if List.mem_assoc key overrides
+         then List.assoc key overrides
          else value in
```

Comment:
This could be simplified to `let value = try List.assoc key overrides with Not_found -> value`

Reply:
This is code from the commits that I cherry-picked from the master branch. To keep rebasing simple,  I'd rather not change it. I agree that List.mem/List.assoc is an anti pattern.